sdk/Pulumi.Tests/Core/OutputTests.cs

```diff
@@ -961,8 +961,21 @@ public void OutputToStringThrowsWhenSet(string? variableValue)
                 var o1 = CreateOutput(0, isKnown: true);
                 Assert.Throws<InvalidOperationException>(() => o1.ToString());
 
+                // If we don't throw check that we log a warning, and then return the expected warning string.
```

Comment:
I can't comment there, but was the Pulumi submodule updated intentionally in this PR?

Reply:
No I'll fix that.